app/statistics/server/functions/get.js

```diff
@@ -146,5 +148,34 @@ statistics.get = function _getStatistics() {
 	statistics.uniqueOSOfYesterday = Sessions.getUniqueOSOfYesterday();
 	statistics.uniqueOSOfLastMonth = Sessions.getUniqueOSOfLastMonth();
 
+	if (Apps.isEnabled()) {
+		const installedApps = Apps.getManager().get();
+		const activeApps = Apps.getManager().get({ enabled: true });
+		statistics.apps = {
+			totalInstalled: installedApps.length,
+			totalActive: activeApps.length,
+			installedAppsList: installedApps.map((app) => ({
+				id: app.getID(),
+				name: app.getName(),
+				status: app.getStatus(),
+			})),
+		};
+	}
+
+	const integrationsCursor = Integrations.find();
```

Comment:
This got me thinking, is it the `.find()` or the `.fetch()` method that actually goes to the database to retrieve data? 🤔

Because, if it is `.fetch()`, I'd suggest saving its result and then iterating over that, to avoid multiple network requests

Reply:
@sampiaodiego Opinion would be nice 🙃

But does seem like fetching and then filtering might be a bit more efficient then multiple fetches/finds

Reply:
Yes guys, you are right, is the [fetch that goes](https://docs.meteor.com/api/collections.html#Mongo-Collection-find) to the database to retrieve data. My bad, I'll fix. :+1: